### PR TITLE
PROGRAMMES-6560 Directing all non-TV audio to playout in Sounds

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6560

Seemed suspiciously easy. I must be missing something.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by benolot** [comment]
Is that the correct URL? I thought sounds/playspace was `radio/play/live:<SID>`. I might be misunderstanding things though, so feel free to ignore me.

File: src/DsShared/Helpers/StreamableHelper.php [hunk 1]
```diff
@@ -46,7 +44,7 @@ public function shouldStreamViaIplayer(ProgrammeItem $programmeItem): bool
     public function shouldStreamViaPlayspace(ProgrammeItem $programmeItem): bool
     {
         $network = $programmeItem->getNetwork();
-        if ($network && in_array($network->getNid(), $this->playspaceNetworksList) && $this->shouldTreatProgrammeItemAsAudio($programmeItem)) {
+        if ($network && !$network->isTv() && $this->shouldTreatProgrammeItemAsAudio($programmeItem)) {
```
Comment (BjornTwachtmann) [inline src/DsShared/Helpers/StreamableHelper.php]: I think this is probably correct for now. Thinking about it, I wonder what is supposed to happen in the case that a thing doesn't have a network, there's quite a lot of clips like this, and at least 1 episode.

Comment (BjornTwachtmann) [reply src/DsShared/Helpers/StreamableHelper.php]: I've raised it with product. For now this is correct.

**Pull request merged**